src/main/resources/kafka_versions.json

```diff
@@ -0,0 +1,6 @@
+{
+  "kafkaVersions": {
+    "2.8.1": "quay.io/strimzi-test-container/test-container:latest-kafka-2.8.1",
+    "3.0.0": "quay.io/strimzi-test-container/test-container:latest-kafka-3.0.0"
```

Comment:
I really feel we should consider embedding a version number for the contract between the container and `test-containers` in the tag or name of the images. We can't guarantee that we won't make changes in `test-container-images` in the future which are incompatible with the version of `test-container` which we release now. Using a tag like `v1-kafka-3.0.0` allows us to have a `v2-kafka-3.0.0` in the future and it not break existing uses of `test-container`. We may never need it, but having it allows us to to more easily change our minds.

Reply:
I think this is the master version. The release versions would not have `latest` but some version of the images in it.

Reply:
Yes as Jakub mentioned the release version of `test-container-images` would look like this:
```
 "3.0.0": "quay.io/strimzi-test-container/test-container:0.100.x-kafka-3.0.0"
```

So it should be okay or @tombentley?